packages/zoe/src/contractFacet/types.js

```diff
@@ -184,6 +197,14 @@
  * @returns {Amount<any>}
  */
 
+/**
+ * @deprecated Use atomicRearrange instead
```

Comment:
I think [JSDoc does not support deprecating callbacks](https://stackoverflow.com/a/42457901/46040). In VSCode there's no strikethrough like other deprecations. I think because `incrementBy` property itself isn't deprecated. The notice is from line 223. 
<img width="461" alt="Screenshot 2023-06-09 at 7 51 51 AM" src="https://github.com/Agoric/agoric-sdk/assets/21505/94697bff-ead8-4615-be6a-e430634ffdd0">

To mark these two properties as deprecated we'll have to use `.ts` syntax for the types.  Unfortunately, that makes is harder to share them ambiently. We could put them in the global namespace but that's not technically correct.

Alternately, we could define the `ZCFSeat` type using the concrete implementation, which can also mark things deprecated. I'm fine deferring that.

Meanwhile, please revert making `DeprecatedIncrementDecrementBy` as it doesn't do anything :)

Reply:
Thanks for checking this, and looking for other alternatives. It annoyed me that I couldn't find a way to get a strike-through on `incrementBy` and `decrementBy`.